FuntastyKit.podspec

```diff
@@ -1,19 +1,19 @@
 Pod::Spec.new do |s|
   s.name = "FuntastyKit"
-  s.version = "1.2.1"
+  s.version = "1.3.0"
   s.summary = "A collection of Swift utilities and protocols used in our projects."
   s.description = <<-DESC
-     A collection of Swift utilities and protocols used in our projects.
+     MVVM-C architecture, service holder for code injection of services
+     some regularly used UIKit extensions, protocols for simple initialization from XIB files,
+     storyboards and for handling keyboard, hairline and keyboard height constraints
   DESC
   s.homepage = "https://github.com/thefuntasty/FuntastyKit"
   s.license = { :type => "MIT", :file => "LICENSE" }
-  s.author = { "Petr Zvonicek" => "" }
+  s.author = { "Matěj K. Jirásek" => "matej.jirasek@thefuntasty.com" }
```

Comment:
https://www.youtube.com/watch?v=Y3OiIjIvN0A

Reply:
Jen jsem Petra chtěl odstínit od zbytených emailů, ale poladím to do příštího releasu, protože na CocoaPods webu se to zobrazuje jinak než jsem myslel. 😂